otwcode/otwarchive
# AO3-5834 Rename cache expiration config settings

## Issue

https://otwarchive.atlassian.net/browse/AO3-5834

## Purpose

We had a 500 error on the bookmarks page because the config variable had a typo. I changed the names to match those suggested in https://github.com/otwcode/otwarchive/pull/3700#pullrequestreview-324914309 and made sure they were the same across all the files.

I silenced excessive output about indexing in the tests like https://travis-ci.org/otwcode/otwarchive/jobs/622186450#L1707-L1720

## Testing Instructions

Log in as admin, enable the experimental caching setting, and then ensure the latest work and bookmark index pages load without giving a 500 error.

Check Travis output to see if annoying output is gone.
